src/controllers/commands/CommandController.cpp

```diff
@@ -538,6 +539,31 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         return "";
     });
+
+    this->registerCommand(
+        "/streamlink", [](const QStringList &words, ChannelPtr channel) {
+            if (words.size() < 2)
+            {
+                channel->addMessage(makeSystemMessage(
+                    "Usage: /streamlink <channel>, use \"channel\" literally "
+                    "to open current channel in streamlink."));
```

Comment:
If we allow `/streamlink` to be called without parameters, the literal `channel` parameter could go away too, making this usage message a bit more clear